pages/PreviewPage/index.js

```diff
@@ -0,0 +1,65 @@
+import { StyledButton } from "@/components/StyledButton";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import Header from "../../components/Header";
+import PreviewPicture from "../../components/PreviewPicture";
+import { saveAs } from "file-saver";
+import { css } from "styled-components";
+
+const imageSrc = "https://i.imgur.com/8nUXl6s.jpeg/";
```

Comment:
Don't you want to pass `imageSrc`, `imageName` and `imageID` as props or rename the `index.js` file as `[id].js` to make the routing dynamic? Otherwise it always shows the same picture.